.github/workflows/npm-audit-fix.yml

```diff
@@ -11,8 +11,11 @@ name: Npm audit fix and compile
 on:
   workflow_dispatch:
   schedule:
-    # At 2:30 on Tuesday
-    - cron: '30 2 * * 2'
+    # At 2:30 on Sundays
```

Comment:
I'd prefer Tuesday. It's the day we do releases. Running this on Sundays means my Monday notification page is filled with audit fixes.